api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java

```diff
@@ -2046,18 +2046,6 @@ private void enableMCHGainNewborn() {
         }
     }
 
-    private void enableANCProgram() {
-        configureBasicProgramDashboard(PihEmrConfigConstants.PROGRAM_ANC_UUID);
-    }
```

Comment:
may be a fools errand, but I tried grouping all the "enable*Program" methods to the same section of the code, sorted alphabetically.